strongbox-web-core/src/main/java/org/carlspring/strongbox/jtwig/extensions/HumanReadableSizeConversionFunction.java

```diff
@@ -0,0 +1,63 @@
+package org.carlspring.strongbox.jtwig.extensions;
+
+import java.util.Collection;
+import java.util.Collections;
+
+import org.jtwig.functions.FunctionRequest;
+import org.jtwig.functions.JtwigFunction;
+
+import com.google.common.collect.Lists;
+
+/**
+ * 
+ * @author ankit.tomar
+ *
+ */
+public class HumanReadableSizeConversionFunction implements JtwigFunction
+{
+
+    @Override
+    public String name()
+    {
+        return "humanReadableSize";
+    }
+
+    @Override
+    public Collection<String> aliases()
+    {
+        return Collections.unmodifiableList(Lists.newArrayList("readableSize"));
```

Comment:
No need to use guava here (and the import) :)
```suggestion
        return Collections.unmodifiableList(Arrays.asList("readableSize"));
```

Reply:
This is what I was thinking to initially. :)